source/features/hide-watch-and-fork-count.css

```diff
@@ -1,3 +1,6 @@
-.pagehead-actions :is(#repo-network-counter, #repo-notifications-counter) {
+.rgh-hide-watch-and-fork-count .pagehead-actions :is(
```

Comment:
i don't know if the `.pagehead-actions` is needed here. it wasn't there before the [PR](https://github.com/refined-github/refined-github/pull/5197) that both fixed the issue because of github's redesign, and made the feature non-disableable.